ci/unit_tests/wawCommons/getFilesAtPath_test.py

```diff
@@ -4,11 +4,123 @@
+        filePath1 = os.path.join(self.dataBasePath, 'directory1', 'file1.test')
+        filePath2 = os.path.join(self.dataBasePath, 'directory2', 'file2.test')
+        pathList = wawCommons.getFilesAtPath([filePath1, filePath2])
+
+        assert len(pathList) == 2
+        absPath1 = os.path.abspath(filePath1)
+        assert absPath1 in pathList
+        absPath2 = os.path.abspath(filePath2)
+        assert absPath2 in pathList
+
+
+    def test_filePathAndDirectoryCombination(self):
+        """Tests if getFilesAtPath when there is a combination of file and directory in input parameter."""
+        filePath = os.path.join(self.dataBasePath, 'directory1', 'file1.test')
+        directory = os.path.join(self.dataBasePath, 'directory2')
+        pathList = wawCommons.getFilesAtPath([filePath, directory])
+
+        testFile1 = os.path.abspath(filePath)
+        assert testFile1 in pathList
+        testFile2 = os.path.abspath(os.path.join(directory, "file2.test"))
+        assert testFile2 in pathList
+
+
+    def test_emptyPatterns(self):
+        """Tests if getFilesAtPath returns no files when no patterns supplied."""
+        directory = self.dataBasePath
+        pathList = wawCommons.getFilesAtPath([directory], [])
+        assert len(pathList) == 0
+
+    def test_filePatternsInDirectory(self):
+        """Tests if files returned from getFilesAtPath can be filtered by patterns parameter."""
+        directory = os.path.join(self.dataBasePath, 'directory1')
+        pathList = wawCommons.getFilesAtPath([directory], ["*.test"])
+
+        testFile1 = os.path.abspath(os.path.join(directory, "file1.test"))
+        assert testFile1 in pathList
+        testFile2 = os.path.abspath(os.path.join(directory, "fileA.ext"))
+        assert testFile2 not in pathList
+
+
+    def test_filePatternsOnFilePath(self):
+        """Tests if file returned from getFilesAtPath can be filtered by patterns parameter."""
+        filePath1 = os.path.join(self.dataBasePath, 'directory1', 'file1.test')
+        filePath2 = os.path.join(self.dataBasePath, 'directory1', 'fileA.ext')
+        pathList = wawCommons.getFilesAtPath([filePath1, filePath2], ["*.test"])
+
+        assert len(pathList) == 1
+        absPath1 = os.path.abspath(filePath1)
+        assert absPath1 in pathList
+        absPath2 = os.path.abspath(filePath2)
+        assert absPath2 not in pathList
+
+
+    def test_multiplePattterns(self):
+        """Tests if mutiple patterns in getFilesAtPath behave like there is operator OR between them."""
+        directory = os.path.join(self.dataBasePath, 'directory1')
+        pathList = wawCommons.getFilesAtPath([directory], ["*.test", "*"])
+
+        testFile1 = os.path.abspath(os.path.join(directory, "file1.test"))
+        assert testFile1 in pathList
+        testFile2 = os.path.abspath(os.path.join(directory, "fileA.ext"))
+        assert testFile2 in pathList
```

Comment:
More tests (suggestions):
- see comment above for inspiration
- test nonExistentFolder
- add more paths to nonExistent* tests to see if it does not skip the rest when finds the first non existing path
- multiple patterns on multiple paths

Reply:
Tests were added